library/snackbar/snackbar.css

```diff
@@ -0,0 +1,90 @@
+.snackbar {
+  visibility: hidden;
+  min-width: 250px;
+  text-align: center;
+  border-radius: 2px;
+  padding: 0.5rem;
+  margin: 0.5rem;
+  position: fixed;
+  z-index: 1;
+
+  bottom: 30px;
+}
+
+.snackbar-center {
+  margin-left: auto;
+  margin-right: auto;
+  left: 0;
+  right: 0;
+  max-width: 250px;
+}
+
+.snackbar-right {
+  right: 0;
+}
+
+.snackbar-default {
+  background-color: var(--primary-color-light);
+  color: #0000;
```

Comment:
var for colors in this file

Reply:
Since it was a generic color, didn't feel like adding it to var, but yes good catch will do!